core/app/models/spree/google_feed_setting.rb

```diff
@@ -0,0 +1,19 @@
+module Spree
+  class GoogleFeedSetting < Base
+    belongs_to :store, class_name: 'Spree::Store', foreign_key: 'spree_store_id'
+
+    validates :store, presence: true
+
+    def enabled_keys
+      keys = []
+
+      attributes.each do |key, value|
+        if value.instance_of?(TrueClass)
```

Comment:
```suggestion
        if value == true
```
I would just use `if value == true` or `if value` syntax, since it's simpler/easier to read.

Reply:
Yeah, I honestly don't remember why I wrote it this way/